src/ahbicht/expressions/expression_resolver.py

```diff
@@ -0,0 +1,46 @@
+"""
+This module makes it possible to parse expressions with all its subexpressions
+for example ahb_expressions with condition_expressions or condition_expressions with packages.
+"""
```

Comment:
```suggestion
Parsing expressions that are nested into other expressions is refered to as "resolving".
"""
```
vllt kann man im docstring noch erwähnen, dass wir das als "resolven" bezeichnen damit der term (auch außerhalb des glossars und der function unten so halb formal eingeführt ist)